lib/vsc/administration/ldapsync.py

```diff
@@ -114,7 +114,8 @@ def sync_altered_accounts(self, last, dry_run=True):
             except HTTPError:
                 logging.error("No corresponding UserGroup for user %s" % (account.vsc_id,))
                 continue
-            gecos = ensure_ascii_string(account.person.gecos)
+            # we avoid storing the real name belong to the account in the ldap
+            gecos = str(account.vsc_id)
```

Comment:
what about eg email?
i would also add a slapd.conf to limit access for default user credentials to whol ebunc of attrs

Reply:
Sure, for us that's already the case. It's a dedicated user for postfix that can read mail. But that's not relevant for this script?

The only other piece that is directly pointing to the person behind the account is the `instituteLogin`. If nobody uses that, it can also be removed. 

In general we already use client and server certificates and limit access to the fields.